src/Commands/stubs/listener.stub

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace $NAMESPACE$;
+
+use $EVENTNAME$;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class $CLASS$
+{
+    /**
+     * Create the event listener.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Handle the event.
+     *
+     * @param $EVENTSHORTENEDNAME$ $event
+     * @return void
+     */
+    public function handle($EVENTSHORTENEDNAME$ $event)
```

Comment:
Eventname is the fully qualified class name (plus namespace).. having to use EVENTSHORTENEDNAME is more like a personal sentiment.. I would fix that though